src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/InterventionController.kt

```diff
@@ -27,8 +27,10 @@ class InterventionController(
     @RequestParam(name = "pccRegionIds", required = false) pccRegionIds: List<String>?,
     @RequestParam(name = "allowsFemale", required = false) allowsFemale: Boolean?,
     @RequestParam(name = "allowsMale", required = false) allowsMale: Boolean?,
+    @RequestParam(name = "minimumAge", required = false) minimumAge: Int?,
```

Comment:
would it make more sense to use `UInt` for these params?

Reply:
Will do

Reply:
Decided against this due to it being an experimental feature